src/ITensorAutoHOOT/batch_contraction.jl

```diff
@@ -15,17 +15,43 @@ struct Executor
   feed_dict::Dict
 end
 
+struct NetworkCache
```

Comment:
Could you explain what kinds of objects this stores?

For example, does it store a set of optimal einsum expressions, or a set of tensor networks (networks of ITensors)?

Reply:
Sure, just added some comments for this struct. NetworkCache.net_sums stores lists of AutoHOOT nodes (einsum expressions), and NetworkCache.index_dict maps each AutoHOOT node to the index of the ITensor tensor in the input array.

Reply:
Thanks!